src/test/groovy/com/github/benmanes/gradle/versions/ConstraintsSpec.groovy

```diff
@@ -131,4 +131,70 @@ final class ConstraintsSpec extends Specification {
     result.output.contains('No dependencies found.')
     result.task(':dependencyUpdates').outcome == SUCCESS
   }
+
+  def "Does not show updates for an gradle constraint added in 7.3.2"() {
```

Comment:
Can you add a test showing the update for log4j if defined in the build file for normal project dependencies?

Reply:
done